sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_arg_dim.cpp

```diff
@@ -12,5 +12,7 @@
 
 // -ffp-model=precise is added to not depend on compiler defaults.
 
+// XFAIL: gpu
```

Comment:
Please add
// XFAIL-TRACKER: CMPLRLLVM-63710
so you don't increase number of xfail without tracker

Reply:
Thanks for the suggestion! I have made the change. 
These two tests are not supported in the `igc-dev` CI test because the commit in IGC has not yet been integrated into the driver. I will remove the XFAIL once the IGC change has been pull downed.

Reply:
hmm, looking at the branch that is used for `igc-dev` here https://github.com/intel/llvm/blob/sycl/devops/dependencies-igc-dev.json

it should include your changes for runtime dimension support:
https://github.com/intel/intel-graphics-compiler/commits/6ee988a5948c6aaecbfe00605f52694c55cf1052/IGC/Compiler/Optimizer/OpenCLPasses/JointMatrixFuncsResolutionPass/JointMatrixFuncsResolutionPass.cpp

So, tests should be passing already with `igc-dev` in intel/llvm CI.
What am I missing?

Reply:
I’ve noticed that the `igc_dev` CI test is not reflecting the IGC commits it's supposed to be testing. It seems to still be using an older version of IGC, likely the version set for the regular IGC. 

I’m experiencing the same issue with another LLVM PR (https://github.com/intel/llvm/pull/15880), where my IGC commit should already be integrated into `igc_dev`, but the tests `get_coordinate_ops.cpp` is still failing due to errors that should have been addressed by that IGC commit. As a temporary approach, I’ve added XFAIL to the test while we wait for this issue to be resolved or until the regular IGC is updated to include the IGC changes.

lease let me know if this approach will work or if you have any suggestions! Thanks

Reply:
@YixingZhang007 ,
"while we wait for this issue to be resolved" - does it mean someone is working on resolving the issue? If not, could you please at least submit the issue to https://github.com/intel/llvm/issues, so that it is reported, and we can assign someone to work on it?
@jsji , @sarnex , FYI

I'm OK with adding XFAIL, since we have this problem.

Reply:
yeah lets make sure this is tracked and someone is investigating

Reply:
Looks like there are issues with the IGC `packaging` ? They started to build and generate `libigc.so.2` instead of `libigc.so.1` now? Please follow up with IGC team to see whether it is intended?

Reply:
Also IGC CI builds started to fail ~ 1 week ago again. https://github.com/intel/intel-graphics-compiler/actions/runs/12034730678 Need someone to follow up with them to fix .

Reply:
They are working on something called `IGC 2` I think, so maybe they enabled it by default?

Reply:
https://github.com/intel/intel-graphics-compiler/commit/4617a3ee0c36a76ceef8e40216218ffe2196381e seems intended to me

Reply:
I have created an issue https://github.com/intel/llvm/issues/16251 regarding this.

Reply:
If so, we need to make sure we softlink to libigc.so to libigc.so.2 for igc-dev now.

Reply:
yeah im gonna look at the issue and try to see what's going on, i gotta fix igc-dev containers first